engine/src/main/java/org/terasology/engine/persistence/serializers/EventSerializer.java

```diff
@@ -154,7 +154,7 @@ public Class<? extends Event> getEventClass(EntityData.Event eventData) {
                 }
             }
             if (metadata == null) {
-                logger.warn("Unable to deserialize unknown event with id: {}", eventData.getType());
+                logger.atWarn().addArgument(() -> eventData.getType()).log("Unable to deserialize unknown event with id: {}");
```

Comment:
`getType()` should be O(1) so impact neglectable
```suggestion
                logger.warn("Unable to deserialize unknown event with id: {}", eventData.getType()); //NOPMD
```